.circleci/config.yml

```diff
@@ -340,6 +340,9 @@ jobs:
       verify_localization:
         type: boolean
         default: true
+      scheme:
```

Comment:
Can we use `configuration` here? As `scheme` is used to be a different thing in terms of Xcode projects.

Reply:
thx, it was a typo I promise 🙂